pkg/app/validate_app.go

```diff
@@ -65,6 +64,11 @@ func NewValidator(config ValidatorConfig) (*Validator, error) {
 
 			ProjectName: project.Name(),
 			Provider:    config.Provider,
+
+			// `ValidateResourcesExist` influences ConfigMaps and Secrets existence
+			// part of the validation. When `true` the `managed-by` labels
+			// are respected, otherwise these labels do not influence validation.
+			ValidateResourcesExist: true,
```

Comment:
```suggestion
			// `EnableManagedByLabel` enables skipping checks for
			// ConfigMap and Secret existence when the `giantswarm.io/managed-by`
			// label is present. This is used when app CRs are managed
			// with gitops tools like flux.
			EnableManagedByLabel: true,
```

Damnit 😢 proving once again that #NamingIsHard I think there is a problem with the flag name.

When used in app-operator this needs to be set to false but the resources are still validated. This flag is actually about whether the managed-by label is active.

Reply:
Ok, it should be done, thank @rossf7 for catching it!